.gitignore

```diff
@@ -45,3 +45,6 @@ spatialdata-sandbox
 
 # version file
 _version.py
+
+# other
+node_modules/
```

Comment:
I saw it as well, what is this for? just out of curisotiy

Reply:
yeah I have in the multiple table PR as well

Reply:
not sure tbh, I think it came from the pre-commits